recipe/meta.yaml

```diff
@@ -38,7 +33,7 @@ requirements:
     - jpeg                                   # [build_platform != target_platform]
     - icu                                    # [build_platform != target_platform]
     - libpng                                 # [build_platform != target_platform]
-    - flatbuffers <2.0.6                     # [build_platform != target_platform]
+    - flatbuffers <2                        # [build_platform != target_platform]
```

Comment:
This will be interesting as this differs from the global pinning (2)

Reply:
I didn't notice that 😰 

Should I revert back to 2.0.6 everywhere?

Reply:
They pin <2 in their setup.py

Reply:
https://github.com/tensorflow/tensorflow/blob/d8ce9f9c301d021a69953134185ab728c1c248d3/tensorflow/tools/pip_package/setup.py#L78-L80

Reply:
https://github.com/tensorflow/tensorflow/issues/55740

Reply:
the global pin isn't apparent to me actually, I cannot find it in the automatically generated yaml config files... are we sure we are supposed to have a global pin?

Reply:
Also, it will be good if we reach a conclusion on this part...